demo-notebooks/testing_instructions.txt

```diff
@@ -0,0 +1,40 @@
+A tested version of Search & Download has been created, named "Main_Site_Search_&_Download-Python-new-testable-example.ipynb"
+Modifications were made to handle our example dataset and improve error messages, but the core features of the notebook remain unchanged. 
+
+After cloning the repository, navigate to the directory "demo-notebooks"
+
+You must first install Pytest and nbval, using:
+
+```pip install pytest```
+
+then 
+
+```pip install nbval```
+
+After confirming that the testing framework has been installed, you need to provide a valid token from your ESS-DIVE account (new one needed every 24 hours):
+
+- Open "secrets.txt": enter your authentication token and save the file 
+
+NOTE: Authentication tokens for the same account vary between the specific site used (e.g. ess-dive production site vs. data-stage)
```

Comment:
If the data-stage notebook is removed from the PR, consider combining lines 18-20 into something like this:
> NOTE: The testing notebook ("Main_Site_Search_&_Download-Python-new-testable-example.ipynb") points to our production endpoint at `https://data.ess-dive.lbl.gov/`. Please ensure that you retrieve your authentication token from the correct endpoint.